cmd/pulumi-test-language/tests/l2_component_call_simple_liftedreturn.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2025, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	"github.com/pulumi/pulumi/cmd/pulumi-test-language/providers"
+	"github.com/pulumi/pulumi/pkg/v3/display"
+	"github.com/pulumi/pulumi/pkg/v3/resource/deploy"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/resource"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/resource/plugin"
+	"github.com/stretchr/testify/require"
+)
+
+func init() {
+	LanguageTests["l2-component-call-simple-liftedreturn"] = LanguageTest{
+		Providers: []plugin.Provider{&providers.ComponentProviderReturnScalar{}},
+		Runs: []TestRun{
+			{
+				Assert: func(l *L,
+					projectDirectory string, err error,
+					snap *deploy.Snapshot, changes display.ResourceChanges,
+				) {
+					RequireStackResource(l, err, changes)
+
+					// We expect the following resources:
+					//
+					// 0. The stack
+					//
+					// 1. The builtin Pulumi provider (used for hydrating resource references)
```

Comment:
As mentioned above I think we can strip out resource references, the spare method etc. from this test. We just need one custom resource with one method being called into one output whose return value we are non-object-ing.